app/src/App.js

```diff
@@ -33,23 +33,20 @@ class App extends Component {
 
   onCompileFromRemix() {
     const plugin = this
-    extension.call('editor', 'getCurrentFile', [], function (error, result) {
+    plugin.result = {}
+    extension.call('editor', 'getCurrentFile', [], (error, result) => {
       console.log(error, result)
-      plugin.setState({
-        placeholderText: result[0]
-      })
+      plugin.result.placeholderText = result[0]
       extension.call('editor', 'getFile', result, (error, result) => {
         console.log(result)
-        plugin.setState({
-          vyper: result[0]
-        })
-        console.log(plugin.state.vyper)
-        plugin.compile(plugin.onCompileSucceeded, plugin.onCompileFailed)
+        plugin.result.vyper = result[0]
+        plugin.setState(plugin.result)
```

Comment:
What does this code mean?

Reply:
L42 is enough?

Reply:
ah~~~

Reply:
I misunderstood
sorry

Reply:
`placeholderText` has already been set at L39, then `plugin.result` is like `{placeHolderText: "hoge", vyper: "hoge"}` at L43.
Both state will update.

Reply:
no problem!